src/fu-engine.c

```diff
@@ -353,7 +354,7 @@ fu_engine_set_emulator_phase(FuEngine *self, FuEngineEmulatorPhase emulator_phas
 gboolean
 fu_engine_emulation_load_phase(FuEngine *self, FuEngineEmulatorPhase emulator_phase, GError **error)
 {
-	return fu_engine_emulator_load_phase(self->emulation, emulator_phase, error);
+	return fu_engine_emulator_load_phase(self->emulation, emulator_phase, 0, error);
```

Comment:
good place to use a define for the the 0 to mean something like `EMULATION_FIRST_WRITE`

Reply:
Agree, fixed. Thanks!